index.bs

```diff
@@ -1060,12 +1072,12 @@ associated.
 
 <div dfn-type="dict-member" dfn-for="AuthenticatorSelectionCriteria">
     :   <dfn>attachment</dfn>
-    ::  If this memeber is [=present|present=], eligible authenticators are filtered to only authenticators attached with the 
+    ::  If this memeber is [=present|present=], eligible authenticators are filtered to only authenticators attached with the
```

Comment:
spelling: s/memeber/member/ ...unless this was actually calling for bear memes, in which case we should make this a MUST imo.